### PR TITLE
adding mask to hide credit card number

### PR DESCRIPTION
**Pull request by jonataswebjump**

### CODE REVIEW

File: Factories/LoggerFactoryInterface.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ *
+ * @author      Webjump Core Team <dev@webjump.com>
+ * @copyright   2016 Webjump (http://www.webjump.com.br)
+ * @license     http://www.webjump.com.br  Copyright
+ *
+ * @link        http://www.webjump.com.br
+ */
+interface LoggerFactoryInterface
+{
+	public static function make($message);
```
Comment (rodrigowebjump) [inline Factories/LoggerFactoryInterface.php]: Fix indentation.

File: Factories/LoggerFactory.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+     * @return string
+     */
+    public static function makeLogRequest(\Psr\Http\Message\RequestInterface $request)
+    {
+        $headers = "";
+        foreach ($request->getHeaders() as $name => $values) {
+            $headers .= $name .": " . implode(", ", $values);
+        }
+        $bodyString = preg_replace('/\"creditCard\"\:\{\"cardNumber\"\:\"(.*?)(\d{4})\",\"holder\"\:(.*?)\,\"expirationDate\"\:(.*?),\"securityCode\"\:(.*?)\,\"saveCard\"\:(.*?)\,\"brand\"\:(.*?)\}/is',
+            '"creditCard":{"cardNumber":"************$2","holder":$3,"expirationDate":"$4","securityCode":"***","saveCard":"$6","brand":$7}',
```
Comment (rodrigowebjump) [inline Factories/LoggerFactory.php]: Replace by checking all string at once for me not looks like a safety way. If one fails, no one will be replaced.
I think that should be one replace per time (array) in one call or another most effective and guaranteed way.

**Review by rodrigowebjump — CHANGES_REQUESTED** [review]

File: Factories/LoggerFactoryInterface.php (hunk 1 above)
Comment (jonataswebjump) [reply Factories/LoggerFactoryInterface.php]: Done.

File: Factories/LoggerFactory.php (hunk 2 above)
Comment (jonataswebjump) [reply Factories/LoggerFactory.php]: Done on last commit. Please review again.

**Review by leandro-rosa — APPROVED** [review]

**Pull request merged**